lib/dynamic.js

```diff
@@ -31,7 +31,7 @@ export default function dynamicComponent (p, o) {
 
       this.LoadingComponent = options.loading ? options.loading : () => (<p>loading...</p>)
       this.ssr = options.ssr === false ? options.ssr : true
-
+      this.compName = options.component ? options.component : undefined
```

Comment:
We don't need this expression. Use `options.componentName` instead. It's clear and does the job.